Lib/importlib/abc.py

```diff
@@ -70,6 +70,15 @@ class ResourceLoader(Loader):
 
     """
 
+    def __init__(self):
+        import warnings
+        warnings.warn('importlib.abc.ResourceLoader is deprecated in '
+                      'favour of supporting resource loading through '
+                      'importlib.resources.abc.ResourceReader.',
```

Comment:
ResourceReader is deprecated in favour of TraversableResources

Reply:
Nice catch! I added this in another PR recently and forgot to update it here. I'll fix this later today if you don't beat me to it ;)

Reply:
ah I opened a PR

Reply:
https://github.com/python/cpython/pull/128859

Reply:
Thanks! Though, part of that is already covered by https://github.com/python/cpython/pull/128601

Reply:
Ah ok hmm, I'm afk for a while so I'm happy for my PR to be closed to make way for a narrower PR to fix just this issue and a followup for the wider change?

Reply:
@tomasr8 https://github.com/python/cpython/pull/128865 is probably next priority?

Reply:
> Ah ok hmm, I'm afk for a while so I'm happy for my PR to be closed to make way for a narrower PR to fix just this issue and a followup for the wider change?

I'm on my phone ATM so it's harder to check, but my PR only addresses the docs, not the deprecation message, so you don't need to close yours (just maybe keep only the message change) :)

Reply:
I have either merged or turned on auto-merging on your two PRs. Thanks for the quick fixes!